shared/src/test/scala/squants/thermal/TemperatureSpec.scala

```diff
@@ -106,52 +121,63 @@ class TemperatureSpec extends FlatSpec with Matchers {
     x.inFahrenheit should be(Fahrenheit(32))
     x.in(Kelvin) should be(Kelvin(273.15))
     x.in(Fahrenheit) should be(Fahrenheit(32))
+    (x.in(Rankine) - Rankine(491.67)).value < 0.000000000001 should be(right = true)
```

Comment:
I wonder if this is a good place to use the `=~` operator with a given tolerance